pkg/collector/prometheus_collector.go

```diff
@@ -478,6 +481,8 @@ func (p *PrometheusCollector) updateNodeMetrics(wg *sync.WaitGroup, ch chan<- pr
 			prometheus.CounterValue,
 			1,
 			collector_metric.NodeCPUArchitecture,
+			components.GetSourceName(),
+			platform.GetSourceName(),
```

Comment:
@rootfs , I see that we have "rapl" hardcoded for source for `.node*JoulesTotal,` shouldn't we use the `components.GetSourceName()` instead? 

```
kepler_node_core_joules_total{instance="",mode="dynamic",package="0",source="rapl"} 15301.647
```